sklearn/preprocessing/tests/test_data.py

```diff
@@ -851,6 +852,64 @@ def test_robust_scaler_iris_quantiles():
     assert_array_almost_equal(q_range, 1)
 
 
+def test_quantile_normalizer_iris():
+    X = iris.data
+    normalizer = QuantileNormalizer()
+
+    X_trans = normalizer.fit_transform(X)
+    # FIXME: one of those will drive to precision error
+    # in the interpolation
+    # assert_array_almost_equal(np.min(X_trans, axis=0), 0.)
```

Comment:
I checked yesterday for while and there is nothing wrong with our code.
`f(min(X))` of the interpolated function do not want to return 0.
The issue should come from `numpy.interp`

This is working on the toy :D 
I will try to sort out the issue with the CI error coming from different numpy version I think.